src/List/Dl.php

```
@@ -70,6 +71,20 @@ public function dt(string|Stringable $content): static
+    /**
+     * Cleans up the output after rendering the block element.
+     *
+     * Removes excessive consecutive newlines from the rendered output to ensure clean HTML structure.
+     *
+     * @param string $result Rendered HTML output.
+     *
+     * @return string Cleaned HTML output with excessive newlines removed.
+     */
+    protected function afterRun(string $result): string
+    {
+        return parent::afterRun(LineBreakNormalizer::normalize($result));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\LineBreakNormalizer' in method 'afterRun'. (undefined)

(StaticAccess)